### PR TITLE
Help Page Update 4

### PR DESCRIPTION
**Pull request by antares1470**
### Description of the Change

Made some updates to some of views help pages to reflect changes made for the upcoming v2.4 release.
Help Pages Updated:
- Notes View
- Layers View
- Conversation View
- Quality Control View
- Plugin Reporter
- Import from File
- Import from Database

### Why Should This Be In Core?

Help pages for things in Core

### Benefits

Up to date help

### Applicable Issues

#340

### CODE REVIEW

**Review by arcturus2 — APPROVED** [review]

File: CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html [hunk 1]
```diff
@@ -79,12 +79,16 @@ <h1>Plugin Reporter</h1>
...
-            <li><em>General</em> - Plugins which don't fit into any specific category.</li>
+            <li><em>Create</em> - Plugins which create graphs, nodes, and transactions.</li>
+            <li><em>Modify</em> - Plugins that change the graph in some way.</li>
+            <li><em>Select</em> - Plugins which change what is selected on the graph.</li>
+            <li><em>Import</em> - Plugins which add data from other sources to the graph. Generally found in the Data Access View.</li>
+            <li><em>Clean</em> - Plugins which clean up the existing data on your graph. Found in the Data Access View.</li>
+            <li><em>Enrich</em> - Plugins which add more information to the existing nodes and transactions on your graph. Found in the Data Access View.</li>
+            <li><em>Extend</em> - Plugins which add more information with new nodes and transactions to your graph. Found in Data Access View.</li>
+            <li><em>Experimental</em> - Plugins which perform experimental features in Constellation.</li>
+            <li><em>Analytic</em> - Plugins which perform some analytic calculation based on information on the graph. Found in the Analytic View</li>
```
Comment (aldebaran30701) [inline CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html:90]: Isn't there a UTILITY tag also?

**Review by aldebaran30701 — COMMENTED** [review]
Looks great, just want to clarify on the one comment.

File: CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html (hunk 1 above)
Comment (aldebaran30701) [reply CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html:90]: also Welcome ?

**Review by Delphinus8821 — APPROVED** [review]

File: CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html (hunk 1 above)
Comment (antares1470) [reply CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html:90]: @aldebaran30701 There are a few tags missing I've intentionally omitted. This list (according to the line leading in) is only a sample of the tags one could see. I could change this to the full list we expect if you think it is worthwhile but else I'm happy to leave it as is

**Review by sol695510 — APPROVED** [review]

File: CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html (hunk 1 above)
Comment (aldebaran30701) [reply CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html:90]: Ohh so the plugin reporter will not have access to all tags? Will plugin reports that are tagged as WELCOME appear when viewing all reports? Would seem off to me that if you can see them in the reporter, you should be able to filter with them (or omit the reports when viewing all) Interested in thoughts on that assumption

Comment (antares1470) [reply CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html:90]: You can see and filter on all the tags available, what I meant was I was only listing a few of the tags available on the help page

Comment (aldebaran30701) [reply CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html:90]: For completeness I'd say its better to have every tag, I feel like if I was confused why there isn't all tags + explanations, a user could also be confused.

Comment (arcturus2) [reply CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html:90]: @antares1470 I would prefer the full list please.

Comment (antares1470) [reply CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/docs/plugin-reporter-view.html:96]: Done

**Review by aldebaran30701 — APPROVED** [review]

**Pull request merged**